### ISSUE TITLE
ATS fails if GWT run separate from GWF 

### ISSUE THREAD

**Issue by RyanConway91** [MODFLOW-USGS/modflow6#899]
I am running a SS GWF model model and then using the cbbs and heads to run a GWT model. If I do not use ATS, the model runs. 

If I use ATS, I get the following error:

Solution 1 did not converge for stress period 1 and time step 1
 Solution Group 1 did not converge for stress period 1 and time step 1
 Failed solution for step 1 and period 1 will be retried using time step of    10.00000    

ERROR REPORT:

  1.     PERIOD NUMBER IN BUDGET FILE DOES NOT MATCH PERIOD NUMBER IN
     TRANSPORT MODEL.  IF THERE IS MORE THAN ONE TIME STEP IN THE BUDGET FILE
     FOR A GIVEN STRESS PERIOD, BUDGET FILE TIME STEPS MUST MATCH GWT MODEL
     TIME STEPS ONE-FOR-ONE IN THAT STRESS PERIOD.

UNIT ERROR REPORT:

  1. ERROR OCCURRED WHILE READING FILE
     'C:\offline\tb3\cal2_add_addtl_flow_prds_add_trns_uzt_adtv\tb3.cbb'

**Comment by langevin-usgs** [comment]
Are there multiple stress periods and time steps in the GWF CBB and HDS files?  Or is there only one time step and one stress period in the CBB and HDS files?

**Comment by RyanConway91** [comment]
There are multiple stress periods in the CBB and HDS files

**Comment by langevin-usgs** [comment]
If the GWF Model has multiple time steps in a stress period, then the GWT Model must have the exact same number of time steps in the stress period.  This will likely not be the case if you are using ATS, which is why you are getting that error.  You should run the GWF Model with only one time step per stress period.  I think ATS will then work for you.

**Comment by RyanConway91** [comment]
Running the GWF model with only one timestep in each stress period solved the issue. I suppose a "hack" to get around this limitation for a GWF SP with multiple TS would be to break up the single SP into multiple SPs. 
Or it would be nice if ATS also worked when there is more than 1 TS in a SP.

**Comment by langevin-usgs** [comment]
Yes, I think I agree with you about this.  If there are more than one time steps in the flow model (and flows are being provided from a previous GWF simulation), then we require that the GWT simulation have the exact same number of time steps.  But I suppose it might be possible to support ATS in this case to ensure that a transport time step never lands beyond the end of a flow time step by reducing the time step length to land at the end of the flow time step.  Let's leave this issue open as a reminder.
